solr/bin/solr

```diff
@@ -1155,6 +1155,11 @@ else
   REMOTE_JMX_OPTS=()
 fi
 
+# Do not use the java security manager when running Java 24+
+if (( JAVA_VER_NUM >= 24 )) ; then
```

Comment:
could just combine this condition into line 1164.  Ideally this changes the "true" in the default interpretation of that env var to a "false".  But it's fine.

Reply:
I'll keep it separate since this is already tested.